src/routes/sessions/[id]/+page.svelte

```diff
@@ -34,6 +34,7 @@
 	import ButtonDelete from '$lib/components/ButtonDelete.svelte';
 	import Metadata from '$lib/components/Metadata.svelte';
 	import Head from '$lib/components/Head.svelte';
+	import { beforeNavigate } from '$app/navigation';
```

Comment:
I usually try to keep the 3rd-party library imports at the top of the file (such as this one from Svelte).
I was hoping prettier would sort them automatically.